pkg/cmd/cli/cmd/newapp.go

```diff
@@ -271,15 +274,56 @@ func setLabels(labels map[string]string, result *newcmd.AppResult) error {
 	return nil
 }
 
+// isInvalidTriggerError returns true if the given error is
+// a validation error that contains 'trigger' in its error message.
+// This error is returned from an older server when
+// it doesn't recognize a newer trigger type.
```

Comment:
"This error is returned from older servers that consider the presence of unknown trigger types to be an error"?